test/components/students/Add_To_Watchlist_Button.spec.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
You can remove this file. We've just updated to React 18, and removed all the enzyme tests.